tests/tests/fv3_ccpp_gfs_v15p2_RRTMGP

```diff
@@ -0,0 +1,85 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v15.2 w/ RRTMGP compiled with 32-bit dynamics test
```

Comment:
I know this is really picky, but the dynamics are not compiled in 32bit for your tests (see COMPILE lines in `rt.conf`, they don't have `32BIT=Y`). Best to simply remove "compiled with 32-bit dynamics"